Makefile

```diff
@@ -16,7 +16,9 @@ e2e-test: build-image
 runtime-samples:
 	cd runtime-samples && make
 
+rust-unit-tests:
+	cd extractor && make unit-tests
+
```

Comment:
OK so we need to run it separately right?

Reply:
yes, it'll be part of the container image used for OpenShift CI: https://github.com/openshift/release/pull/57340/files#diff-d4ad928f8a8dcf0f971c2ff43f2aca7b420c1379b4f7864682c25ac241b4cd01R17-R34